### PR TITLE
Domain autojoin settings

### PR DESCRIPTION
**Pull request by flvndvd**
This PR is the concluding piece that unlocks the ability for administrators to set up auto-joining of workspaces for new users. This feature applies to new users who sign-up with a Google Workspace verified email domain, provided it matches the verified email domain of the workspace.

WorkspaceAutoJoinSettings

### CODE REVIEW

**Review by spolu — APPROVED** [review]
Code LGTM let's have @Duncid review UX wording if possible :pray:

**Comment by flvndvd** [comment]
> Code LGTM let's have @Duncid review UX wording if possible 🙏

Implemented based on the design from https://www.figma.com/file/QOxHwppG64GtPocpDhS6Y7/Product?type=design&node-id=1909-83163&mode=design&t=BejwunBkcYtBDcsl-0. Happy to take another pass with @Duncid.

**Comment by spolu** [comment]
Surprised by the choice of having 2 buttons and not a big fan of Google Sign-On in the title. But not stongly held.

LGTM :+1:

File: front/pages/w/[wId]/members/index.tsx [hunk 1]
```diff
@@ -99,80 +107,66 @@ export default function WorkspaceAdmin({
...
+          onClose={() => {
+            setIsActivateAutoJoinOpened(false);
+          }}
+          domain={domain}
+          owner={owner}
+        />
+        {workspaceVerifiedDomain && (
+          <Page.Vertical gap="sm">
+            <Page.H variant="h5">
+              Auto-join Workspace with Google Single Sign-On
```
[the remark this answers is not in the document]
Comment (flvndvd) [reply front/pages/w/[wId]/members/index.tsx]: Fixed.

File: front/pages/w/[wId]/members/index.tsx [hunk 2]
```diff
@@ -99,80 +107,66 @@ export default function WorkspaceAdmin({
...
+                onClick={() => {
+                  if (!isUpgraded(plan)) {
+                    setShowNoInviteLinkPopup(true);
+                  } else {
+                    setIsActivateAutoJoinOpened(true);
+                  }
+                }}
+              />
+              <Button
+                label="De-activate Auto-join"
```
[the remark this answers is not in the document]
Comment (flvndvd) [reply front/pages/w/[wId]/members/index.tsx]: Fixed.

**Comment by flvndvd** [comment]
> Surprised by the choice of having 2 buttons and not a big fan of Google Sign-On in the title. But not stongly held.
> 
> LGTM 👍

@spolu updated based on your comments. GIF updated in the PR description.

**Pull request merged**